todo_app/templates/index.html

```diff
@@ -3,28 +3,97 @@
 
 {% block content %}
   <div class="jumbotron">
-    <h1 class="display-4">To-Do App</h1>
+    <h1 class="display-6">To-Do App</h1>
     <p class="lead">Just another to-do app.</p>
+    <form action="/additem" method="post" class="form-example">
+      <div class="form-example">
+        <label for="name">New To-Do item: </label>
+        <input type="text" name="new-todo" required>
+        <button type="submit" name ="Submit">Add</button>
+      </div>
+    </form>
   </div>
-
   <div class="row">
     <div class="col-md-12">
-      <h2>Items</h2>
+      <h3>To-Do</h3>
       <ul class="list-group mb-4">
-        {% for each_item in items_list %}
-          <li>#{{each_item.id}}: {{ each_item.title }} - {{each_item.status }}</li>
+        {% for each_todo in todo_list %}
+        <table style="width:100%">
+          <colgroup>
+            <col style="width: 50%">
+            <col style="width: 25%">
+            <col style="width: 25%">
+          </colgroup>
+          <td>
+            <li>Task: {{each_todo.name}} | <!--ID: {{each_todo.id}} |--> <b>PENDING</b></li>
```

Comment:
I like the fact you're hiding the ID here - it's not really providing any useful info to the user - so I'd suggest we commit and remove it entirely!
```suggestion
            <li>Task: {{each_todo.name}} | <b>PENDING</b></li>
```